packages/x-license/src/licenseErrorMessageUtils.ts

```diff
@@ -14,31 +14,31 @@ function showError(message: string[]) {
 
 export function showInvalidLicenseError() {
   showError([
-    'Material-UI X: Invalid license.',
+    'MUI X: Invalid license.',
```

Comment:
@oliviertassinari Should the "MUI" prefix be used for license errors?

```suggestion
    'MUI: Invalid license.',
```

x other cases in this file.